NCEP/graphcast_job_cloud.sh

```diff
@@ -55,7 +55,7 @@ echo "Execution time for gdas_utility.py: $execution_time seconds"
 start_time=$(date +%s)
 echo "start runing graphcast to get real time 10-days forecasts for: $curr_datetime"
 # Run another Python script
-python3 run_graphcast.py -i source-gdas_date-"$curr_datetime"_res-0.25_levels-13_steps-40.nc -o "$curr_datetime"
+python3 run_graphcast.py -i source-gdas_date-"$curr_datetime"_res-0.25_levels-13_steps-1.nc -o "$curr_datetime"
```

Comment:
Will this run graphcast 1 step? The message (line 56) said the it's 10 day dorecast

Reply:
that should be 2 that shows the size of inputs, let me update it

Reply:
Now updated!